inference-engine/src/offline_transformations/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 
 target_link_libraries(${TARGET_NAME} PUBLIC ${NGRAPH_LIBRARIES} inference_engine_transformations
-                                     PRIVATE openvino::itt)
+                                     PRIVATE openvino::itt ngraph::reference)
```

Comment:
I didn't find any reference implementations in your code, did I miss something?

Reply:
Needed by CoordinateTransform class.